rdwatch/scoring/tasks/animation_export.py

```diff
@@ -601,24 +601,24 @@ def create_site_animation_export(
         celery_id=task_id,
         arguments=settings,
     )
-    # try:
-    site_export.celery_id = task_id
-    site_export.save()
-    file_path, name = create_animation(self, site_evaluation_id, settings)
-    if file_path is False and name is False:
-        logger.warning('No Images were found')
-        site_export.delete()
-        return
-    site_export.name = name
-    with open(file_path, 'rb') as file:
-        site_export.export_file.save(name, File(file))
-    site_export.save()
-    if os.path.exists(file_path):
-        os.remove(file_path)
-    # except Exception as e:
-    #     logger.warning(f'Error when processing Animation: {e}')
-    #     if site_export:
-    #         site_export.delete()
+    try:
+        site_export.celery_id = task_id
+        site_export.save()
+        file_path, name = create_animation(self, site_evaluation_id, settings)
+        if file_path is False and name is False:
+            logger.warning('No Images were found')
+            site_export.delete()
+            return
+        site_export.name = name
+        with open(file_path, 'rb') as file:
+            site_export.export_file.save(name, File(file))
+        site_export.save()
+        if os.path.exists(file_path):
+            os.remove(file_path)
+    except Exception as e:
+        logger.warning(f'Error when processing Animation: {e}')
+        if site_export:
+            site_export.delete()
```

Comment:
Same question here. If an animation fails to generate, would it be better to get an exception in Sentry instead of suppressing it?
```suggestion
    try:
        site_export.celery_id = task_id
        site_export.save()
        file_path, name = create_animation(self, site_evaluation_id, settings)
        if file_path is False and name is False:
            logger.warning('No Images were found')
            site_export.delete()
            return
        site_export.name = name
        with open(file_path, 'rb') as file:
            site_export.export_file.save(name, File(file))
        site_export.save()
        if os.path.exists(file_path):
            os.remove(file_path)
    finally:
        logger.warning(f'Error when processing Animation: {e}')
        if site_export:
            site_export.delete()
```

Reply:
> I'm open to some way to log this to sentry without causing the task to exit.

Yes, this is possible - see https://github.com/ResonantGeoData/RD-WATCH/pull/529#pullrequestreview-2441268767